src/ray/raylet/local_object_manager.h

```diff
@@ -165,8 +166,8 @@ class LocalObjectManager {
   void SpillObjectsInternal(const std::vector<ObjectID> &objects_ids,
                             std::function<void(const ray::Status &)> callback);
 
-  /// Release an object that has been freed by its owner.
-  void ReleaseFreedObject(const ObjectID &object_id);
+  /// Remove the owner of an object that has been freed by its owner.
```

Comment:
Can you keep the comment the object is released by this function? (but mention it happens when there's 0 owner)